modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -1442,6 +1447,54 @@ describe("Token Voting Client", () => {
         const token = await client.methods.getToken(pluginAddress);
         expect(token).toBe(null);
       });
+
+      it("Should check if a ERC20 is compatible with governance and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc20Token = await deployErc20();
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
```

Comment:
Why does this method still have the old name? Shouldn't it be called `isTokenVotingComaptibleToken` ?